CODEOWNERS

```
@@ -1,8 +1,7 @@
 # Where component owners are known, add them here.
 
-/oneTBB/src/tbb/ @pavelkumbrasev
+/oneTBB/src/tbb/ @kboyarinov
 /oneTBB/src/tbb/ @dnmokhov
-/oneTBB/src/tbb/ @sarathnandu
 /oneTBB/include/oneapi/tbb/parallel_* @pavelkumbrasev
```

If @dnmokhov do not mind of course:)
```suggestion
/oneTBB/include/oneapi/tbb/parallel_* @kboyarinov
/oneTBB/include/oneapi/tbb/paralle_* @dnmokhov
```